src/errors.rs

```diff
@@ -1,12 +1,17 @@
 error_chain! {
+    foreign_links {
+        Io(::std::io::Error)#[allow(missing_docs)];
```

Comment:
It seems a bug of error chain, I can't put document comment in here. I'd like to create more specific error kinds, rather than just `wrap std::io::Error`.

Reply:
Ok, let's leave it as is and open an issue (after the PR is merged) to refactor it later.

Reply:
the issue: https://github.com/killercup/cargo-edit/issues/320